I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script00.rpy", line 500, in script call
    call dungeon
  File "game/script00.rpy", line 581, in script
    call screen dungeon_run(True)
  File "renpy/common/000statements.rpy", line 457, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "game/dungeonBattle.rpy", line 250, in event
    self.callScene("quitScreen")
  File "game/dungeonBattle.rpy", line 235, in callScene
    self.toggleActive()
  File "game/dungeonBattle.rpy", line 128, in toggleActive
    ui.visible = reactivate
UnboundLocalError: local variable 'reactivate' referenced before assignment

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script00.rpy", line 500, in script call
    call dungeon
  File "game/script00.rpy", line 581, in script
    call screen dungeon_run(True)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/ast.py", line 1635, in execute
    self.call("execute")
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/ast.py", line 1653, in call
    renpy.statements.call(method, parsed, *args, **kwargs)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/statements.py", line 144, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 457, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/exports.py", line 2311, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/ui.py", line 247, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/core.py", line 2149, in interact
    repeat, rv = self.interact_core(preloads=preloads, **kwargs)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/core.py", line 2750, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/layout.py", line 846, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/layout.py", line 846, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/layout.py", line 846, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/screen.py", line 626, in event
    rv = self.child.event(ev, x, y, st)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/layout.py", line 846, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/display/motion.py", line 827, in event
    rv = d.event(ev, cx, cy, st)
  File "game/dungeonBattle.rpy", line 250, in event
    self.callScene("quitScreen")
  File "game/dungeonBattle.rpy", line 235, in callScene
    self.toggleActive()
  File "game/dungeonBattle.rpy", line 128, in toggleActive
    ui.visible = reactivate
UnboundLocalError: local variable 'reactivate' referenced before assignment

Linux-3.11.0-26-generic-x86_64-with-debian-wheezy-sid
Ren'Py 6.18.3.761
Summoner Sweetheart 0.9
